packages/wallet-service/tests/txProcessor.test.ts

```diff
@@ -515,300 +93,3 @@ describe('NFT metadata updating', () => {
     spyCreateOrUpdate.mockRestore();
   });
 });
-
-test.skip('receive token creation tx', async () => {
```

Comment:
We also need to add this one, we're adding token information into the `tokens` table, but not testing it.

Maybe an integration test with token creation simulations.